support.go

```diff
@@ -5,6 +5,9 @@ import "encoding/json"
 type Support struct {
 	Files    []SupportFile    `yaml:"files,omitempty" json:"files,omitempty" validate:"dive"`
 	Commands []SupportCommand `yaml:"commands,omitempty" json:"commands,omitempty" validate:"dive"`
+	// Timeout is the seconds to wait for custom support commands to complete.
+	// If zero, the default timeout will be applied.
+	Timeout int `yaml:"timeout,omitempty" json:"timeout,omitempty"`
```

Comment:
Should this be a [UintString](https://github.com/replicatedhq/libyaml/blob/9498acb60a72fb62292faf1f4240b250ec2e9d55/uint_string.go) to support template values?